grub-efi/Dockerfile.x86_64

```diff
@@ -11,6 +11,10 @@ RUN wget --progress=dot:mega ftp://ftp.gnu.org/gnu/grub/grub-${GRUB_VERSION}.tar
 
 WORKDIR grub-${GRUB_VERSION}
 
+# For 2.12 a patch is needed, see:
+# https://git.savannah.gnu.org/cgit/grub.git/commit/?id=b835601c7639ed1890f2d3db91900a8506011a8e
+RUN if [ "$GRUB_VERSION" = 2.12 ]; then echo "depends bli part_gpt" > grub-core/extra_deps.lst && GRUB_AUTORECONF=1; fi
+
```

Comment:
It is a bit ugly that we ended up copying this everywhere. However the Dockerfiles were already copies so I think it is the simplest solution and good enough for this repository :ok_hand: